h/accounts/schemas.py

```diff
@@ -85,6 +86,13 @@ def unblacklisted_username(node, value, blacklist=None):
         raise colander.Invalid(node, msg)
 
 
+def required_acceptance(node, value):
```

Comment:
I'm not thrilled with this, either, but I found that writing a custom validator, in the end, was the only way to get things accomplished, sanely.